packages/mdc-icon-toggle/mdc-icon-toggle.scss

```diff
@@ -19,7 +19,6 @@
 @import "@material/animation/functions";
 @import "@material/ripple/common";
 @import "@material/ripple/mixins";
-@import "@material/theme/mixins";
```

Comment:
Don't remove this. `mdc-theme-prop` is still used in this file (test with `npm run build` for now)